uber/site_sections/summary.py

```diff
@@ -110,6 +110,14 @@ def personalized_badges(self, out, session):
                                                 Attendee.amount_extra >= c.SUPPORTER_LEVEL).order_by(Attendee.full_name).all():
             out.writerow(['', 'Supporter', a.badge_printed_name or a.full_name])
 
```

Comment:
This is indeed a good example of the kind of thing you can do, but I assume we're not actually merging this in with these test files and such, right?